source/application/AMDatamanAppController.cpp

```diff
@@ -750,11 +759,11 @@ bool AMDatamanAppController::startupRegisterDatabases()
 	success &= AMDbObjectSupport::s()->registerClass<AMXESScan>();
 	success &= AMDbObjectSupport::s()->registerClass<AM2DScan>();
 	success &= AMDbObjectSupport::s()->registerClass<AM3DScan>();
-	success &= AMDbObjectSupport::s()->registerClass<AMXRFScan>();
```

Comment:
You can get rid of this because you're just using AMScan instead of a specific AMXRFScan type, right?